### PR TITLE
Add util function to identify duplicate clients in HMIS

### PR DESCRIPTION
**Pull request by gigxz**
## Description

Util function to write duplicate clients to a file. This could be useful to run again after an initial HMIS migration, when there might be many dups coming from the old system.

## Type of change
- [ ] Bug fix
- [x] New feature (adds functionality)
- [ ] Code clean-up / housekeeping
- [ ] Dependency update

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: drivers/hmis/lib/hmis_data_cleanup/util.rb [hunk 1]
```diff
@@ -179,6 +187,93 @@ def self.write_project_unit_summary!(filename: 'hmis_project_summary.csv')
       end
     end
 
+    def self.write_potential_duplicates(filename: 'hmis_potential_duplicates.csv', variant: 'all', full_name: true)
```
Comment (eanders) [inline drivers/hmis/lib/hmis_data_cleanup/util.rb]:
```suggestion
    def self.write_potential_duplicates(filename: 'hmis_client_potential_duplicates.csv', variant: 'all', full_name: true)
```

File: drivers/hmis/lib/hmis_data_cleanup/util.rb [hunk 2]
```diff
@@ -179,6 +187,93 @@ def self.write_project_unit_summary!(filename: 'hmis_project_summary.csv')
...
+          [client.id, comparison_attrs]
+        end.to_h
+
+      rows = []
+      destination_id_to_source_ids.each do |dest_id, source_ids|
+        row = {
+          WarehouseID: dest_id,
+        }
+
+        8.times do |idx|
```
Comment (eanders) [inline drivers/hmis/lib/hmis_data_cleanup/util.rb]: Is this just because 8 is a big number?

**Review by eanders — APPROVED** [review]
Looks good and useful to me!  Thanks.

File: drivers/hmis/lib/hmis_data_cleanup/util.rb (hunk 2 above)
Comment (gigxz) [reply drivers/hmis/lib/hmis_data_cleanup/util.rb]: That was the largest number of duplicates in the environment where I was running this

**Pull request merged**